python/requirements-wheel-build.txt

```diff
@@ -1,16 +1,5 @@
 cython>=0.29.11
-setuptools>=58
+oldest-supported-numpy
```

Comment:
same here, this file shouldn't be change AFAIK

Reply:
It would be better if we could rely on `oldest-supported-numpy` here.

Reply:
Ah, so we use this file to manually create the "build environment", from where we call `python setup.py bdist_wheel`. Then I suppose we could indeed use `oldest-supported-numpy` here. 

Now, I think the "proper" solution would be to stop using `python setup.py bdist_wheel`, and use `python -m build` instead (https://pypa-build.readthedocs.io/en/stable/). Then that would create the build environment based on what is already specified in pyproject.toml, and it would follow the same workflow as when you build a wheel when installing an sdist. 
Although that brings up an interesting question: arrow-cpp also requires numpy to build. Should this be the same version? Or that should also follow the idea of "build against oldest supported"? (in which case we need to keep numpy anyway in this requirements file)

Reply:
Let's keep the requirements files for now and defer the `pypa-build` question to another issue.